src/Rules/Functions/ParameterCastableToStringRule.php

```diff
@@ -67,7 +60,6 @@ public function processNode(Node $node, Scope $scope): array
 		if (
 			!in_array($functionName, $checkAllArgsFunctions, true)
 			&& !in_array($functionName, $checkFirstArgFunctions, true)
```

Comment:
I kept these two groups together, since the only difference is which parameters are checked.